js/app.js

```diff
@@ -320,6 +320,10 @@ $(function(){
           this.removeLastConditionFromActor(Actors.selectedActor()); break;
         case 114:  // 'r'
           this.rotateConditions(Actors.selectedActor()); break;
+        case 113:  // 'q'
+          this.switchActorListLeft(); break;
+        case 81: // 'Q'
+          this.switchActorListRight(); break;
```

Comment:
Let's bind this behavior to `\` and have it toggle on press, rather than bound to a pair of keys